src/ol/interaction/mousewheelzoominteraction.js

```diff
@@ -119,3 +129,20 @@ ol.interaction.MouseWheelZoom.prototype.doZoom_ = function(map) {
   this.startTime_ = undefined;
   this.timeoutId_ = undefined;
 };
+
+
+/**
+ * Disable using the mouse's location as an anchor when zooming
+ */
+ol.interaction.MouseWheelZoom.prototype.disableMouseAnchor = function() {
+  this.lastAnchor_ = null;
+  this.useAnchor_ = false;
+};
+
+
+/**
+ * Enable using the mouse's location as an anchor when zooming
+ */
+ol.interaction.MouseWheelZoom.prototype.enableMouseAnchor = function() {
+  this.useAnchor_ = true;
+};
```

Comment:
What are `disableMouseAnchor` and `enableMouseAnchor` for? They are not exportable (no `@api` annotation), nor are they use internally. Can we do without them?